features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/modules/statistics/usage.jag

```diff
@@ -350,9 +356,14 @@ function getFirstAccessTime(providerName, server) {
     var usage = [];
     var usageout = [];
     try {
-        usage = apiProvider.getFirstAccessTime(providerName, server);
+        if(usageClient==null){
```

Comment:
fixed in commit: https://github.com/wso2/carbon-apimgt/commit/9619da2271b49f37ef93456b6f8a8ac235f685ea